### ISSUE TITLE
Reload ipset_file without restarting AdGuardHome

### ISSUE THREAD

**Issue by user9931** [AdguardTeam/AdGuardHome#5676]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

### Description

adguardhome is not very fast thing to restart completely, especially on low-end devices (unlike dnsmasq) so please consider some way to reload options from ipset_file without restart (maybe on SIGHUP)

**Comment by Freebase394** [comment]
Yes, Its completely true! On my Raspbery Pi 3b+ its take ~1/3 minutes to restart.
 💯
